cmd/ketch/app_info.go

```diff
@@ -59,6 +57,20 @@ type appInfoContext struct {
 	Table       string
 }
 
+type appInfoOutput struct {
+	AppInfoContext appInfoContext
```

Comment:
how about having yaml and json tags for `AppInfoContext` and `Deployments`?

Reply:
Added. I added struct tags to builder and env types too.